### ISSUE TITLE
Candidate lists

### ISSUE THREAD

**Issue by myersnews** [TheLens/elections#349]
Senate (bold=frontrunner for coloring purposes and default short list; determined by polling)
Beryl Billiot
**Charles Boustany (R)**
**Foster Campbell (D)**
Joseph Cao (R)
Thomas P. Clements (L)
Donald “Crawdaddy” Crawford (R)
David Duke (R)
Derrick Edwards (D)
**Caroline Fayard (D)**
**John Fleming (R)**
Le Roy Gillam (L)
Troy Hebert 
**John Kennedy (R)**
Gary Landrieu (D)
Bob Lang Jr. (other party)
Rob Maness (R)
Kaitlin Marone 
Charles Marsala (R)
Vinny Mendoza (D)
Abhay Patel (R)
Joshua Pellerin (D)
Gregory Taylor Jr. (other party)
Arden Wells 
Peter Williams (D)

President (bold-show in minimized view)
**Hillary Clinton (D)
Jill Stein (Green Party)
Gary Johnson (L)
Donald Trump (R)**

Darrell Castle (other party)
Evan McMullin (other party)
Laurence Kotlikoff (other party)
Tom Hoefling (other party)
Princess Jacob (other party)
Gloria La Riva (other party)
Jerry White (other party)
Alyson Kennedy (other party)
Chris Keniston (other party)

**Comment by myersnews** [comment]
@ThomasThoren do you need this at all?

**Comment by ThomasThoren** [comment]
Yeah, this information is stored in `data/candidates.json` and is used by both the front and back ends.
